src/tz390.java

```diff
@@ -1216,6 +1224,20 @@ public  class  tz390 {
 //           Alternate syntax is ;*Short which will generate only H, L, E, NH, NL, NE rather than the full set
 //           This syntax allows us to override the default mnemonics for generated instructions.
 //           Omission of mnemon in the override implies that no  mnemonic exists for that mask.
+// Example1: definition of assembler directive AIF: "--=AIF,203,--"                                                                                                    #485
```

Comment:
Observation/question. The example has -- for opcode and op_trace_type. Nothing wrong. However, Remark 1 above (not part of your changes) says "op_type and op_trace_type are entered as -- for assembler directives. Should this be changed to "opcode and op_trace_type are entered as -- for assembler directives"???

Reply:
The format of entries is documented as:
> opcode=mnemonic,op_type,op_trace_type[;overrides]

the table entry given in example 1 is "--=AIF,203,--"
So AIF is the mnemonic, 203 is the op_type. opcode and op_trace_type are both specified as dashes --

In my reading this is consistent with the current wording of remark 1. 
Yet I am open to a different way of wording the information if that reduces the risk of the comments being misunderstood or misinterpreted. The aim of these comments is to avoid confusion, rather than create it ...